codingtovictory/day6/takeHomeChallenge/index.html

```diff
@@ -0,0 +1,25 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="FirstA">
```

Comment:
Classes are meant for many things whereas id's are meant for one thing. It looks like you are trying to target one thing here so it should be an `id`. Both work just got to know when to use one vs the other @Bs-Sec @OnkurLal